### ISSUE TITLE
goofys not work and not print log in wsl(Windows Subsystem Linux) Ubuntu 18.04

### ISSUE THREAD

**Issue by Allen2Git** [kahing/goofys#443]
I downloaded goofys binary and run in Windows 2019 wsl(Windows Subsystem Linux) Ubuntu 18.04. It's not worked and print error message:
2019/08/11 02:43:20.681328 main.FATAL Unable to mount file system, see syslog for details
But unlucky, there's also no output in /var/log/syslog .
My questions are:
1. How can Goofys support in wsl ubuntu 18.04 in Windows Server 2019?
2. Why there's not error log in /var/log/syslog?
Thanks a lot.

**Comment by yuvisara** [comment]
Try -f to run the process in the foreground which will print the logs.

**Comment by Allen2Git** [comment]
Thanks yuvisara.

I tried with -f parameters like below:
wsl-user@windows-wsl:~$ sudo ./goofys -f  s3-temp-driver /mnt/goofys/
shown below: 

2019/08/12 15:29:20.708697 main.FATAL Mounting file system: Mount: mount: running fusermount: exit status 1
stderr:
fusermount: fuse device not found, try 'modprobe fuse' first

Then I tried sudo modprobe fuse, got:

ec2-user@EC2AMAZ-TAE8VA7:~$ sudo modprobe fuse
modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/4.4.0-17763-Microsoft/modules.dep.bin'
modprobe: FATAL: Module fuse not found in directory /lib/modules/4.4.0-17763-Microsoft

From https://github.com/microsoft/WSL/issues/2869 I can see, no fuse support in wsl now.

Thanks.
